packages/url_launcher/url_launcher_windows/pubspec.yaml

```diff
@@ -24,5 +24,5 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pigeon: ^5.0.1
+  pigeon: ^9.2.4
```

Comment:
Pigeon is a code generator, so updates to it need to update the generated code. Running `update-dependency --pub-package=pigeon --packages/url_launcher_windows` via [the repo tooling](https://github.com/flutter/packages/blob/main/script/tool/README.md) will do the right thing.

(Generally we want to update to the latest Pigeon, but if you really need 9.2.4 for some reason you can add `--version=9.2.4`)

Reply:
Main reason I chose 9.2.4 was that it is the version that is used by other url_launcher. Going to the latest for this package works too. Of course it is possible to rev up pigeon across all plugins, but I tried to minimize the scope of the change.

Reply:
> Pigeon is a code generator, so updates to it need to update the generated code. Running `update-dependency --pub-package=pigeon --packages/url_launcher_windows` via [the repo tooling](https://github.com/flutter/packages/blob/main/script/tool/README.md) will do the right thing.

This hasn't been addressed; the generated code update is still not part of the PR.